.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 before_install:
-  - pip install commentjson
+  - pip install commentjson PyYAML
```

Comment:
why do we use both here?

in verify_files.py itself, you don't use commentjson either:
https://github.com/Cockatrice/Magic-Spoiler/pull/99/files#diff-3068ab58e9b7dcfddfdb3288c3c07890